components/lib/calendar/BaseCalendar.vue

```diff
@@ -196,10 +196,6 @@ export default {
             type: String,
             default: null
         },
-        id: {
```

Comment:
Was been dummy prop only for computed d_id (for a11y) in Calendar.vue: https://github.com/primefaces/primevue/pull/5567/files#diff-29f5ad83722c320c17555abd43c32dc15aa18759ca1894c22c6e1ea53d365bbaL548